openstack/cronus/templates/maillog/deployment.yaml

```diff
@@ -0,0 +1,76 @@
+{{- if .Values.maillog.enabled }}
+{{- if .Values.maillog.deployment.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: maillog
+  {{- if .Values.maillog.annotations }}
+  annotations:
+    {{ toYaml .Values.maillog.annotations | indent 4 }}
+  {{- end  }}
+  {{- if .Values.maillog.labels }}
+  labels:
+    {{ toYaml .Values.maillog.labels | indent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ default .Values.maillog.deployment.replicas }}
+  strategy:
+{{- if eq "RollingUpdate" .Values.maillog.updateStrategy }}
+    rollingUpdate:
+      maxSurge: {{ .Values.maillog.maxSurge }}
+      maxUnavailable: {{ .Values.maillog.maxUnavailable }}
+{{- end }}
+    type: {{ .Values.maillog.updateStrategy }}
+  selector:
+    matchLabels:
+      name: maillog
+      instance: maillog
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include "cronus/templates/maillog/_config.yaml.tpl" . | sha256sum }}
+      labels:
+        name: maillog
+        instance: maillog
+    spec:
+      serviceAccountName: maillog
+{{- if .Values.maillog.affinity }}
+      affinity: {{ toYaml .Values.maillog.affinity | nindent 8 }}
+{{- end }}
+      containers:
+        - name: maillog
+          image: "{{ .Values.maillog.image.name }}:{{ .Values.maillog.image.tag }}"
```

Comment:
You should probably have the `.Values.global.registry` here. So:
`"{{ .Values.global.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"`

Reply:
No, the registry is usually part of the image spec in the secrets.

Reply:
It should be a variable though... for example, in the past the registry url has changed. So by using the global.registry, it is aligning with where the build step in the pipeline pushes to, which can and has changed